<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IssueBook</class>
 <widget class="QWidget" name="IssueBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Issue Book</string>
  </property>
  <widget class="QLabel" name="issue_title_lbl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Book Title: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="title_lineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>381</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>12</number>
    </property>
    <property name="verticalSpacing">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="fname_lbl">
      <property name="text">
       <string>Borrower First Name: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="fname_lineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lname_lbl">
      <property name="text">
       <string>Borrower Last Name: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lname_lineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="mobno_lbl">
      <property name="text">
       <string>Borrower Mobile Number: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="mobno_lineEdit">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="email_lbl">
      <property name="text">
       <string>Borrower Email: </string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="email_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="issue_btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Issue Book</string>
   </property>
  </widget>
  <widget class="QLabel" name="issue_msg_lbl">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>440</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
